### DISCUSSION TITLE
Stuck in exporting gltf

### DISCUSSION THREAD

**Discussion by atul161** [google/filament#6692] [General]
This may sound like a weird question but in my App, we are changing materials, and textures at runtime.
I believe that three js provided an extensive way to perform operations on gltf like adding another mesh, changing material and many more.

What I found is that there is no way I can export rendered glb using filament. As this feature unlock multiple possibilities for android native because through this we natively support multiple things like avatar creation platform and many more. 

@romainguy did i have missed something? 

Although i am very happy with the performances of filament in low end devices.

**Comment by romainguy** [comment]
We do not provide a glTF exporter at this point no. Filament's primary focus is rendering.
